src/aks-preview/HISTORY.md

```diff
@@ -3,6 +3,12 @@
 Release History
 ===============
 
+0.5.64
+++++++
+
+* Add support for updating kubelet identity. Command is
+  * `az aks update --assign-kubelet-identity <kubelelt-identity-resource-id>`
```

Comment:
May I ask what's the difference between `kubelet identity` and `managed identity`?

Reply:
kubelet identity is used by kubelet in agent node, its main purpose is authenticating to ACR to pull image from ACR;
`--enable-managed-identity` or `--assign-identity` refers to control plane identity, which is used by AKS managed master components such as kube-controller-manager, its main purpose to authenticating to Azure to manage Azure resource, for example, when kube-controller-manager find it needs to update load balancer, it will authenticate using control plane identity  to do the operation.

Reply:
As far as I know, there is a (managed or user assigned) identity used by the control plane (i.e., managed cluster) and another one (this kubelet identity) used by the nodes (mainly used for authentication when pulling image from acr).